Atmosphere/RaymarchedClouds/DeferredRaymarchedVolumetricCloudsRenderer.cs

```diff
@@ -286,24 +383,24 @@ void OnPreRender()
                     cloudMaterial.SetVector("reprojectionUVOffset", uvOffset);
                     cloudMaterial.SetFloat("frameNumber", (float)(frame));
 
-                    Vector3 noiseReprojectionOffset = targetCamera.worldToCameraMatrix.MultiplyVector(-intersection.layer.NoiseReprojectionOffset);
-                    Matrix4x4 cloudPreviousV = previousV;
+                    Vector3 noiseReprojectionOffset = currentV.MultiplyVector(-intersection.layer.NoiseReprojectionOffset);
+                    Matrix4x4 cloudPreviousV = prevV;
 
                     // inject upwards noise offset
                     cloudPreviousV.m03 += noiseReprojectionOffset.x;
                     cloudPreviousV.m13 += noiseReprojectionOffset.y;
                     cloudPreviousV.m23 += noiseReprojectionOffset.z;
 
                     cloudMaterial.SetMatrix("currentVP", currentP * currentV);
-                    cloudMaterial.SetMatrix("previousVP", previousP * cloudPreviousV * intersection.layer.OppositeFrameDeltaRotationMatrix);    // inject the rotation of the cloud layer itself
+                    cloudMaterial.SetMatrix("previousVP", prevP * cloudPreviousV * intersection.layer.OppositeFrameDeltaRotationMatrix);    // inject the rotation of the cloud layer itself
 
                     // handle the actual rendering
-                    commandBuffer.SetRenderTarget(useFlipRaysBuffer ? flipRaysRenderTextures : flopRaysRenderTextures, newRaysFlipRT.depthBuffer);
+                    commandBuffer.SetRenderTarget(useFlipRaysBuffer ? flipRaysRenderTextures : flopRaysRenderTextures, newRaysRT[true].depthBuffer); // REVIEW: this always uses flip - why?
```

Comment:
The traditional depth buffer is not written to by my shaders and is not needed as a renderTarget, but I have to provide something to this method otherwise it throws a nullref. Therefore I just provided it that one.